tools-local/regenerate-readme-table.proj

```diff
@@ -1,5 +1,5 @@
 <Project>
```

Comment:
How are building this project? I think you need to add back the subset to eng\subsets.props that was removed during cleanup earlier this year.